webpack.config.js

```diff
@@ -315,7 +315,8 @@ const moduleConfig = function(env, argv) {
                   },
                 ],
                 '@babel/plugin-transform-nullish-coalescing-operator',
-                '@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining',
+                '@babel/plugin-bugfix-v8-spread' +
+                'parameters-in-optional-chaining',
```

Comment:
Not sure this string split will work, since it's missing the `-` between strings. this will end up being:

`babel/plugin-bugfix-v8-spreadparameters-in-optional-chaining` and not `@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining` like we want.